docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob/collect-logs.md

```diff
@@ -0,0 +1,223 @@
+1. Event Hub streams the events to the TaskProducer Azure function, which creates tasks (a JSON object that specifies start and batchsize, container name, blob path) and pushes those tasks to the service bus task queue.
+1. The TaskConsumer Azure function, which is triggered when the service bus receives a new task, reads the append blob, from start byte to batchsize, and sends that data to Sumo. 
+1. The set up also includes failure handling mechanism. For more information about the solution strategy, see [Azure Blob Storage](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/append-blob).
+
+## Step 1. Configure Azure storage account 
+
+In this step, you configure a storage account to which you will export monitoring data for your Azure service. The storage account must be a General-purpose v2 (GPv2) storage account.
+
+If you have a storage account with a container you want to use for this purpose, make a note of its resource group, storage account name and container name, and proceed to [Step 2](#step-2-configure-an-http-source).
+
+To configure an Azure storage account, do the following:
+
+1. Create a new storage account General-purpose v2 (GPv2) storage account. For instructions, see [Create a storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=portal ) in Azure help.
+1. Create a container(Optional) all services in azure create containers automatically. This step is needed only when you are exporting custom logs in some container.
+   * In the Azure portal, navigate to the storage account you just created (in the previous step).
+   * Select **Blobs** under **Blob Service**.
+   * Select **+ Container**.
+   * Enter the Name.
+   * Select **Private** for the Public Access Level.
+   * Click **OK**.
+
+:::note
+Make a note of the container name, as you will need to supply it later.
+:::
+
+## Step 2. Configure an HTTP source
+
+In this step, you configure an HTTP source to receive logs from the Azure function.
+
+1. Select a hosted collector where you want to configure the HTTP source. If desired, create a new hosted collector, as described on [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+1. Configure an HTTP source, as described on [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics). Make a note of the URL for the source, you will need it in the next step.
+
+## Step 3. Configure Azure resources using ARM template
+
+In this step, you use a Sumo-provided Azure Resource Manager (ARM) template to create an Event Hub, three Azure functions, Service Bus Queue, and a Storage Account.
+
+1. Download the [appendblobreaderdeploy.json](https://raw.githubusercontent.com/SumoLogic/sumologic-azure-function/master/AppendBlobReader/src/appendblobreaderdeploy.json) ARM template.
```

Comment:
The link to the appendblobreaderdeploy.json file is broken.

Reply:
once the PR is merged to master (we are waiting for QE go ahead) then only the link becomes active